### ISSUE TITLE
Cannot connect to host error, ssl related

### ISSUE THREAD

**Issue by Giladt2789** [DataDog/datadog-sync-cli#256]
First timer with the tool.
Provided the credentials using a config file (as described in the readme file). Tried to pull the resources from datadog using the command:
``` 
$ datadog-sync import --config config
```
but got the following error:
```
2024-06-24 13:28:03,544 - ERROR - error while validating api key: Cannot connect to host api.datadoghq.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')]
Traceback (most recent call last):
  File "aiohttp/connector.py", line 992, in _wrap_create_connection
  File "asyncio/base_events.py", line 1149, in create_connection
  File "asyncio/base_events.py", line 1182, in _create_connection_transport
  File "asyncio/sslproto.py", line 578, in _on_handshake_complete
  File "asyncio/sslproto.py", line 560, in _do_handshake
  File "ssl.py", line 917, in do_handshake
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "datadog_sync/utils/configuration.py", line 67, in init_async
  File "datadog_sync/utils/configuration.py", line 194, in _validate_client
  File "datadog_sync/utils/configuration.py", line 188, in _validate_client
  File "datadog_sync/utils/custom_client.py", line 30, in wrapper
  File "aiohttp/client.py", line 1194, in __aenter__
  File "aiohttp/client.py", line 578, in _request
  File "aiohttp/connector.py", line 544, in connect
  File "aiohttp/connector.py", line 911, in _create_connection
  File "aiohttp/connector.py", line 1235, in _create_direct_connection
  File "aiohttp/connector.py", line 1204, in _create_direct_connection
  File "aiohttp/connector.py", line 994, in _wrap_create_connection
aiohttp.client_exceptions.ClientConnectorCertificateError: Cannot connect to host api.datadoghq.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1000)')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "datadog_sync/commands/shared/utils.py", line 35, in run_cmd_async
  File "datadog_sync/utils/configuration.py", line 69, in init_async
AttributeError: 'Configuration' object has no attribute '_exit_cleanup'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "datadog_sync/cli.py", line 23, in <module>
  File "click/core.py", line 1157, in __call__
  File "click/core.py", line 1078, in main
  File "click/core.py", line 1688, in invoke
  File "click/core.py", line 1434, in invoke
  File "click/core.py", line 783, in invoke
  File "datadog_sync/commands/_import.py", line 18, in _import
  File "datadog_sync/commands/shared/utils.py", line 19, in run_cmd
  File "asyncio/runners.py", line 194, in run
  File "asyncio/runners.py", line 118, in run
  File "asyncio/base_events.py", line 687, in run_until_complete
  File "datadog_sync/commands/shared/utils.py", line 57, in run_cmd_async
AttributeError: 'NoneType' object has no attribute 'pbar'
[46242] Failed to execute script 'cli' due to unhandled exception!
```

**Steps to reproduce the issue:**
1. Downloaded the [datadog-sync-cli-darwin-arm64](https://github.com/DataDog/datadog-sync-cli/releases/download/0.9.3/datadog-sync-cli-darwin-arm64) file.
2. Created the config file:
```
source_api_key="<API_KEY>"
source_app_key="<APP_KEY>"
source_api_url="https://api.datadoghq.com"
```
3. run the command: 
```
$ datadog-sync import --config config
```

Running the tool on macbook pro, m1 chip

**Comment by skarimo** [comment]
Hello, thanks for opening the issue. Looks like github has switched to using m1 runners and it seems the default cert locations are different between GA and locally. We are looking into fixing this.

In the meantime, here are some workarounds:
 - Set env var `SSL_CERT_FILE`

or 

- Install from source: https://github.com/DataDog/datadog-sync-cli?tab=readme-ov-file#installing-from-source

**Comment by Giladt2789** [comment]
Hi @skarimo 
when i set the env var: SSL_CERT_FILE, what should be its value?

**Comment by skarimo** [comment]
This would be where-ever the certs are located. If using certifi, you can get it by running the python script:
```python
import certifi
certifi.where()
```

It can also be grabbed from openssl, its typically located in its installation directory which can be found using:
```bash
openssl version -d
```
The file should be `cert.pem`

**Comment by Giladt2789** [comment]
Assuming i've ran the:
``` 
$ export SSL_CERT_FILE=path/to/file/cert.pem
$ datadog-sync import --source-api-key="my_key" --source-app-key="my_app_key" --source-api-url="https://api.datadoghq.com"
```
i get the following:
```
2024-06-27 09:50:15,915 - INFO - clients validated successfully
2024-06-27 09:50:15,915 - INFO - Starting import...
2024-06-27 09:50:16,102 - INFO - Getting resources for dashboard_lists
2024-06-27 09:50:16,103 - INFO - Getting resources for dashboards
2024-06-27 09:50:16,104 - INFO - Getting resources for downtime_schedules
2024-06-27 09:50:16,104 - INFO - Getting resources for host_tags
2024-06-27 09:50:16,104 - INFO - Getting resources for logs_indexes
2024-06-27 09:50:16,104 - INFO - Getting resources for logs_indexes_order
2024-06-27 09:50:16,104 - INFO - Getting resources for logs_metrics
2024-06-27 09:50:16,105 - INFO - Getting resources for logs_pipelines
2024-06-27 09:50:16,105 - INFO - Getting resources for logs_pipelines_order
2024-06-27 09:50:16,105 - INFO - Getting resources for logs_restriction_queries
2024-06-27 09:50:16,105 - INFO - Getting resources for metric_percentiles
2024-06-27 09:50:16,105 - INFO - Getting resources for metric_tag_configurations
2024-06-27 09:50:16,106 - INFO - Getting resources for metrics_metadata
2024-06-27 09:50:16,106 - INFO - Getting resources for monitors
2024-06-27 09:50:16,106 - INFO - Getting resources for notebooks
2024-06-27 09:50:16,106 - INFO - Getting resources for powerpacks
2024-06-27 09:50:16,106 - INFO - Getting resources for restriction_policies
2024-06-27 09:50:16,106 - INFO - Getting resources for roles
2024-06-27 09:50:16,106 - INFO - Getting resources for service_level_objectives
2024-06-27 09:50:16,107 - INFO - Getting resources for slo_corrections
2024-06-27 09:50:16,107 - INFO - Getting resources for spans_metrics
2024-06-27 09:50:16,107 - INFO - Getting resources for synthetics_global_variables
2024-06-27 09:50:16,107 - INFO - Getting resources for synthetics_private_locations
2024-06-27 09:50:16,107 - INFO - Getting resources for synthetics_tests
2024-06-27 09:50:16,107 - INFO - Getting resources for teams
2024-06-27 09:50:16,107 - INFO - Getting resources for users
100%|████████████████████████████████████████████████████████████████████████| 26/26 [00:02<00:00, 11.02it/s]
2024-06-27 09:50:18,462 - INFO - Finished getting resources. Successes: 26, Failures: 0, Skipped: 0, Filtered: 0
2024-06-27 09:50:18,462 - INFO - Importing individual resource items
 35%|███████████████████████▋                                           | 1418/4016 [00:03<00:05, 446.92it/s]Usage: datadog-sync [OPTIONS] COMMAND [ARGS]...
Try 'datadog-sync --help' for help.

Error: No such option: -B
100%|████████████████████████████████████████████████████████████████████| 4016/4016 [00:42<00:00, 94.96it/s]
2024-06-27 09:51:00,753 - INFO - Finished importng individual resource items. Successes: 3275, Failures: 0, Skipped: 741, Filtered: 0.
2024-06-27 09:51:00,937 - INFO - Finished import
multiprocessing/resource_tracker.py:123: UserWarning: resource_tracker: process died unexpectedly, relaunching.  Some resources might leak.
```

it seems that this tool isn't quite ready for use..:/

**Comment by skarimo** [comment]
Hello, we have released `0.11.0`  which should resolve the above issues. Feel free to follow up if there are additional questions.
